src/Sentry/IHasTags.cs

```diff
@@ -37,4 +37,19 @@ public static void SetTags(this IHasTags hasTags, IEnumerable<KeyValuePair<strin
             hasTags.SetTag(key, value);
         }
     }
+
+    /// <summary>
+    /// Get tags or null.
+    /// </summary>
+    /// <param name="hasTags"></param>
+    /// <returns></returns>
+    public static IReadOnlyDictionary<string, string>? GetTagsOrNull(this IHasTags hasTags)
```

Comment:
Do we need this method now that we have `InternalTags` again?
```suggestion
    internal static IReadOnlyDictionary<string, string>? GetTagsOrNull(this IHasTags hasTags)
```